x-pack/plugins/secops/server/utils/logger.ts

```diff
@@ -4,20 +4,27 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { Server } from 'hapi';
-
 const LOGGING_TAGS = ['secops'];
 
+// Definition is from:
+// https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/hapi/v16/index.d.ts#L318
+export type HapiLogger = (
+  tags: string | string[],
+  // tslint:disable-next-line:ban-types
```

Comment:
since we don't "own" these types, it makes sense to disable the `tslint` rule here 👍